### ISSUE TITLE
Migrate from FilerField to FilerImageField

### ISSUE THREAD

**Issue by stuaxo** [divio/django-filer#1147]
Hi,
  I'm picked up a legacy project that uses a mix of FilerField instead of FilerImageField for images.
How can I write a migration to move a FilerField to a FilerImageField ?

Would I need make a data migration, so the image dimensions get populated ?

Cheers
S

**Comment by benzkji** [comment]
you would need to introduce a new filer image field. then write a data migration. that collects the files from the existing field, looks for the image (if it is one), and assign the found image to the new field. then remove the old field...and, optionaly, rename the new one...

almost a year, I know ;-)

**Comment by stuaxo** [comment]
Thanks, something I did since then was look into the database.

When I wrote this I assumed that the FilerField wouldn't have the image dimensions, so these would need to be collected from each image, at least on the project I'm using these are all actually present in the database.

Doing this has been a low priority so we still haven't got round to it.

If we ever actually get round to this, I should try and remember to paste something more in detail here.
